docs/requirements.txt

```diff
@@ -1,19 +1,30 @@
-celery
+# https://github.com/sphinx-doc/sphinx/issues/2796
 boto3
+celery
 dogpile.cache
 flask
 flask-login
 flask-migrate
 flask-sqlalchemy
+opentelemetry-api
+opentelemetry-exporter-otlp
+opentelemetry-instrumentation
+opentelemetry-instrumentation-botocore
+opentelemetry-instrumentation-celery
```

Comment:
why do we have to add opentelemetry libraries for this PR?

Reply:
I believe this is only for docs. They are already listed as dependencies in setup.py

Reply:
Exactly. As I added modules, which included parts of the code working with opentelemetry, into documentation, Sphinx was throwing errors about missing opentelemetry libraries.